documentation/API_documentation/IsolatedNetworks_User_Story.md

```diff
@@ -0,0 +1,9 @@
+| **Item**                     | **Details** |
+|------------------------------|-------------|
+| **Summary**               | As a property manager or service owner, I want to create, modify, and delete logically isolated network configurations on operator-managed network access devices (such as gateways or mesh Wi-Fi), so that I can securely segment networks for tenants, guests, or IoT devices, ensuring privacy and easy reconfiguration. |
```

Comment:
```suggestion
| **Summary**               | As a property manager or service owner, I want to create, modify, and delete logically isolated network configurations on API provider-managed network access devices (such as gateways or mesh Wi-Fi), so that I can securely segment networks for tenants, guests, or IoT devices, ensuring privacy and easy reconfiguration. |
```